modules/administration-guide/partials/snip_internal-backup-server-warning.adoc

```diff
@@ -1,7 +1,6 @@
-
 [NOTE]
 ====
-{prod} Operator can automatically setup a backup server inside the same cluster.
-However, this is not recommended for production use.
-If you are still OK to store your backups inside the same namespace as {prod-short} installation, you may skip this section at all.
+{prod} Operator can automatically set up a backup server inside the same cluster; however, it is not recommended for production use
+
+Users willing to back up their data inside the same namespace as {prod-short} installation may skip this section.
```

Comment:
I wouldn't use "willing" as this is more about limitation (user agrees on the limitation to not to manually set up and configure backup server)